hathor/manager.py

```diff
@@ -1044,7 +1054,7 @@ def sync_v2_step_validations(self, txs: Iterable[BaseTransaction], *, quiet: boo
                     try:
                         # XXX: `reject_locked_reward` might not apply, partial validation is only used on sync-v2
                         # TODO: deal with `reject_locked_reward` on sync-v2
-                        assert tx.validate_full(reject_locked_reward=True)
+                        assert tx.validate_full(reject_locked_reward=False)
```

Comment:
```TODO: deal with `reject_locked_reward` on sync-v2``` don't we have to do this before using sync-v2?

Reply:
Sort of, using `reject_locked_reward=False` is needed for correctness because of how the order of validations can happen, as long as at the end the validation that was skipped is re-run for transactions in the mempool.

Reply:
This conversation has been started on another thread. Closing this one.